moshi/src/test/java/com/squareup/moshi/JsonQualifiersTest.java

```diff
@@ -364,21 +364,21 @@ public void toButNoFromJson() throws Exception {
           .hasMessageThat()
           .isEqualTo(
               "No @FromJson adapter for class java.lang.String annotated "
-                  + "[@com.squareup.moshi.JsonQualifiersTest$FooPrefix()]"
+                  + "[@com.squareup.moshi.JsonQualifiersTest.FooPrefix()]"
```

Comment:
Java 19 change...? I don't really know what causes this but I'm also not too concerned